drivers/i3c/Kconfig.stm32

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 EXALT Technologies.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module = I3C_STM32
+module-str = i3c_stm32
+
+source "subsys/logging/Kconfig.template.log_config"
+config I3C_STM32
+	bool "STM32 I3C driver support"
+	depends on DT_HAS_ST_STM32_I3C_ENABLED
+	select USE_STM32_HAL_I3C
+	default y
```

Comment:
Looks like you miss a `select I3C_IBI_WORKQUEUE if I3C_USE_IBI`, if we look at other drivers, otherwise it doesn't compile.

This being said, I wonder why `I3C_IBI_WORKQUEUE` isn't defined as `default y`. @XenuIsWatching maybe you'll be able to tell ?